scripts/backup.pl

```diff
@@ -577,6 +577,7 @@ sub MySQLBackupForMigrateFromOTRS {
 
             # substitutions for changing the character set
             $Line =~ s/DEFAULT CHARSET=utf8/DEFAULT CHARSET=utf8mb4/;    # for CREATE TABLE
+            $Line =~ s/CHARACTER SET .*?\s//;                            # for CREATE COLUMN
```

Comment:
I propose to substitute with a single space, just in case there is only a single whitespace character after the character set.

Reply:
The questions are resolved. The trailing whitespace can be removed, as there is white space before 'CHARACTER' .